contracts/proposals/ccip019-pox-4-stacking.clar

```diff
@@ -0,0 +1,67 @@
+;; TRAITS
+
+(impl-trait .proposal-trait.proposal-trait)
+
+;; ERRORS
+
+(define-constant ERR_PANIC (err u1400))
+
+;; CONSTANTS
+
+(define-constant SELF (as-contract tx-sender))
+(define-constant CCIP_019 {
+  name: "PoX-4 Stacking",
+  link: "",
+  hash: "",
+})
+
+
+;; PUBLIC FUNCTIONS
+
+(define-public (execute (sender principal))
+  (let
+    (
+      (miaId (unwrap! (contract-call? .ccd004-city-registry get-city-id "mia") ERR_PANIC))
+      (nycId (unwrap! (contract-call? .ccd004-city-registry get-city-id "nyc") ERR_PANIC))
+      (miaBalance (contract-call? .ccd002-treasury-mia-mining-v2 get-balance-stx))
+      (nycBalance (contract-call? .ccd002-treasury-nyc-mining-v2 get-balance-stx))
+    )
+
+    ;; enable new treasuries in the DAO
+    (try! (contract-call? .base-dao set-extensions
+      (list
+        {extension: .ccd002-treasury-mia-mining-v3, enabled: true}
+        {extension: .ccd002-treasury-nyc-mining-v3, enabled: true}
+        {extension: .ccd012-rewards-treasury, enabled: true}
```

Comment:
There's no contract with this name? Should match contract name pending updates from earlier comment.

Reply:
added new treasury contract .ccd001-treasury-mia-stx-rewards-v3